Documentation/cloud-config-locations.md

```diff
@@ -15,8 +15,8 @@ On every boot, coreos-cloudinit looks for a config file to configure your host.
 | --- | --- |
 | `/media/configvirtfs/openstack/latest/user_data` | `/media/configvirtfs` mount point with [config-2](config-drive.md#contents-and-format) label. It should contain a `openstack/latest/user_data` relative path. Usually used by cloud providers or in VM installations. |
 | `/media/configdrive/openstack/latest/user_data` | FAT or ISO9660 filesystem with [config-2](config-drive.md#qemu-virtfs) label and `/media/configdrive/` mount point. It should also contain a `openstack/latest/user_data` relative path. Usually used in installations which are configured by USB Flash sticks or CDROM media. |
-| Kernel command line: `cloud-config-url=http://example.com/user_data`. | You can find this string using this command `cat /proc/cmdline`. Usually used in [PXE](https://github.com/coreos/docs/tree/master/os/booting-with-pxe.md) or [iPXE](https://github.com/coreos/docs/tree/master/os/booting-with-ipxe.md) boots. |
-| `/var/lib/coreos-install/user_data` | When you install CoreOS manually using the [coreos-install](https://github.com/coreos/docs/tree/master/os/installing-to-disk.md) tool. Usually used in bare metal installations. |
+| Kernel command line: `cloud-config-url=http://example.com/user_data`. | You can find this string using this command `cat /proc/cmdline`. Usually used in [PXE](https://github.com/flatcar-linux/docs/tree/master/os/booting-with-pxe.md) or [iPXE](https://github.com/flatcar-linux/docs/tree/master/os/booting-with-ipxe.md) boots. |
```

Comment:
The links here should be https://docs.flatcar-linux.org/os/booting-with-pxe/ and https://docs.flatcar-linux.org/os/booting-with-ipxe/